value/src/lib.rs

```diff
@@ -27,6 +27,9 @@ pub use serde_json::Number;
 pub use serializer::{to_value, SerializerError};
 pub use variables::Variables;
 
+#[cfg(feature = "raw_value")]
+pub use value_serde::RAW_VALUE_TOKEN;
```

Comment:
Does not have to be exposed, but it is convenient to have it exposed still. `serde_json` should arguably do the same.